doc/language_guide.md

```diff
@@ -157,115 +157,101 @@ SQLFlow supports specifying various feature columns in the column clause and lab
+
+#### SEQ_CATEGORY_ID
+
+The format is: `SEQ_CATEGORY_ID([DENSE()|SPARSE()|field], BUCKET_SIZE)` `SEQ_CATEGORY_ID` splits the input field by delimiter and returns identity values, represents sequences of integers. In the expression:
+
+- `[DENSE()|SPARSE()|field]` you can specify the field data format using `DENSE`, `SPARSE` or just specify the column name and let SQLFlow to infer the actual format.
+- `BUCKET_SIZE`: the max id of the input sparse tensor, your inputs are integers in the range [0, num_buckets).
+
+#### CATEGORY_HASH
+
+The format is: `CATEGORY_HASH([DENSE()|SPARSE()|field], BUCKET_SIZE)`, represents sparse feature where ids are set by hashing. In the expression:
+
+- `[DENSE()|SPARSE()|field]` you can specify the field data format using `DENSE`, `SPARSE` or just specify the column name and let SQLFlow to infer the actual format.
+- `BUCKET_SIZE` number of bucket of the hashed value.
+
+#### WEIGHTED_CATEGORY
+The format is: `WEIGHTED_CATEGORY([CATEGORY_ID(SPARSE(field,shape,delim,dtype_value,delim_kv,dtype_key))|CATEGORY_HASH(...)])`. Applies weight values to any categorical column like `CATEGORY_ID`. In the expression:
+
+- `WEIGHTED_CATEGORY` accepts one argument, and this argument must be an categorical column expression. The data format of the categorical column must be like `SPARSE(field,shape,delim,dtype_value,delim_kv,dtype_key)`, which will feed the categorical IDs and 
+
+#### EMBEDDING
+
+The format is: `EMBEDDING([CATEGORY_ID(...)|CATEGORY_HASH(...)|WEIGHTED_CATEGORY(...)|col_name], SIZE[, COMBINER, INITIALIZER])`. Represents an embedding layer. In the expression:
+
+- `[CATEGORY_ID(...)|CATEGORY_HASH(...)|WEIGHTED_CATEGORY(...)|col_name]` can be any categorical column as the embedding's input.
+- `SIZE`: the width of the embedding.
+- `COMBINER`: how to reduce if there are multiple entries in a single row, can be `mean`, `sqrtn`, `sum`. Default is `sum`.
+- `INITIALIZER`: weight initializer function, like `truncated_normal_initializer`.
+
+#### INDICATOR
+
+The format is: `INDICATOR([CATEGORY_ID(...)|CATEGORY_HASH(...)|WEIGHTED_CATEGORY(...)|col_name])`. Represents multi-hot representation of given categorical column. The only input argument of `INDICATOR` is any categorical column.
+
+#### CROSS
+
+The format is: `CROSS([column_1, column_2], HASH_BUCKET_SIZE)`. Returns a column for performing crosses of categorical features. In the expression:
+
+- `[column_1, column_2]` a list of two elements respresenting two columns as the input.
+- `HASH_BUCKET_SIZE` number of buckets that the crossed feature will be hashed to.
+
+#### BUCKET
+
+The format is: `BUCKET([DENSE(...)|col_name], BOUNDARIES)`. Represents discretized dense input bucketed by `BOUNDARIES`. In the expression:
+
+- `[DENSE(...)|col_name]` is the input column, can be a numeric column with any shape.
+- `BOUNDARIES` is a list represents the value boundaries, like `[0., 1., 2.]` generates buckets `(-inf, 0.)`, `[0., 1.)`, `[1., 2.)`, and `[2., +inf)`.
```

Comment:
Maybe we can add some examples to illustrate the input and output for each column type.

Reply:
Input examples are in the above table, the users do not need to know the output format.